db/migrate/20181130111723_add_volumes.rb

```diff
@@ -0,0 +1,61 @@
+class AddVolumes < ActiveRecord::Migration[5.1]
+  def change
+    create_table :volumes, :id => :bigserial do |t|
+      t.references :tenant, :type => :bigint, :index => true, :null => false, :foreign_key => {:on_delete => :cascade}
+      t.references :source, :type => :bigint, :index => true, :null => false, :foreign_key => {:on_delete => :cascade}
+      t.references :source_region, :type => :bigint, :index => true, :foreign_key => {:on_delete => :cascade}
+
+      t.string :source_ref
+      t.string :name
+      t.string :status
+      t.bigint :size
+      t.jsonb  :extra
+
+      t.timestamps
+      t.datetime   :archived_on
+      t.datetime   :last_seen_at
+
+      t.datetime   :resource_timestamp
+      t.jsonb      :resource_timestamps, :default => {}
+      t.datetime   :resource_timestamps_max
+
+      t.index [:source_id, :source_ref], :unique => true
+      t.index :archived_on
+    end
+
+    create_table :volume_types, :id => :bigserial do |t|
```

Comment:
@Ladas what are some examples of volume types?  Is this something that could be a string property on a volume or is it really a managed inventory item?  And do we need volume_types initially for cost management?

Reply:
We need volume types for AI ops. So for MIQ we've decided to add separate table, because usually it contains more metadata, so name wasn't enough.

Reply:
btw. these are the types https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html

I think I will also be able to fetch the from the pricing API

Reply:
:+1: thanks